.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -23,5 +23,16 @@ A clear and concise description of what you expected to happen.
 ## Actual Results (including error logs, if applicable)
 A clear and concise description of what actually did happen.
 
+## Reproducible
+   * [ ] Always 
+   * [ ] Sometimes
+   * [ ] Non-Reproducible
+   
+## Version/Tag number
+A clear input of your version/Tag number
```

Comment:
do you think there is a way to provide guidance on how to find the version number generally? or is the best we can do “please provide the version of the product” (it might be - just wanted to ask if you have any ideas)

Reply:
It suppose to be in the logs in some of the products . If not we can always ask for tag number or hash.
In appliance for example we can say: [v5.8.1](https://github.com/conjurinc/appliance/releases/tag/v5.8.1) 7e74eb2